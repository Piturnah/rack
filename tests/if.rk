// program should print "true" and "also true"
// but never print "false"

50 8 - 42 = if
  "true\n" puts

  true if
     false if
       "false\n" puts
     end

     "also true\n" puts
  end

  8 2 = if
    "false\n" puts
  end

end