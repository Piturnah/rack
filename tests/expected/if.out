----STDOUT----
true
also true

----STDERR----
